contentcuration/contentcuration/frontend/shared/views/TipTapEditor/TipTapEditor/components/EditorToolbar.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+
+  <div
+    class="toolbar"
+    role="toolbar"
+    :aria-label="t('textFormattingToolbar')"
+  >
+    <!-- History buttons -->
+    <div
+      role="group"
+      :aria-label="t('historyActions')"
+    >
+      <ToolbarButton
+        v-for="action in historyActions"
+        :key="action.name"
+        :title="action.title"
+        :icon="action.icon"
+        :is-available="action.isAvailable"
+        :should-flip-in-rtl="true"
+        @click="action.handler"
+      />
+    </div>
+
+    <ToolbarDivider />
+
+    <!-- Format dropdown -->
+    <div
+      role="group"
+      :aria-label="t('textFormattingOptions')"
+    >
+      <FormatDropdown />
+    </div>
+
+    <ToolbarDivider />
+
+    <!-- Text formatting -->
+    <div
+      role="group"
+      :aria-label="t('textStyleFormatting')"
+    >
+      <ToolbarButton
+        v-for="action in textActions"
+        :key="action.name"
+        :title="action.title"
+        :icon="action.icon"
+        :is-active="action.isActive"
+        @click="action.handler"
+      />
+    </div>
+
+    <ToolbarDivider />
+
+    <!-- Copy/Paste -->
+    <div
+      role="group"
+      :aria-label="t('copyAndPasteActions')"
```

Comment:
Here's an example line of where we can update to be more aligned with the new composables strategy